mod-infra/src/main/scala/at/forsyte/apalache/infra/passes/options.scala

```diff
@@ -64,53 +229,3 @@ object SMTEncoding {
     case oddEncodingType => throw new IllegalArgumentException(s"Unexpected SMT encoding type $oddEncodingType")
   }
 }
-
```

Comment:
This bits can safely be ignored during review. Followup WIP the implements the options classes can be seen in #2074.